src/pages/devtools/themes/natsuiro/natsuiro.js

```diff
@@ -2253,6 +2253,7 @@
 						.data("masterId", thisNode.drop)
 						.on("dblclick", this.shipDoubleClickFunction)
 						.attr("title", KC3Meta.shipName( KC3Master.ship(thisNode.drop).api_name ))
+						.toggleClass("new_ship", KC3ShipManager.find(w => RemodelDb.remodelGroup(thisNode.drop) === RemodelDb.remodelGroup(w.masterId)).length === 0)

```

Comment:
`KC3ShipManager.find(w => RemodelDb.remodelGroup(thisNode.drop) === RemodelDb.remodelGroup(w.masterId))`

Selects all ships that are in the same remodel group as the dropped ship, eg dupes. Feel free to add a comment above/below to explain it for future maintenance.